hub/api/dataset.py

```diff
@@ -199,8 +200,10 @@ def __init__(
             except Exception as e:
                 try:
                     self.close()
-                except Exception:
-                    pass
+
+                except:
+
+                    raise AuthenticationException
```

Comment:
interesting, authentication error is thrown inside .close() or during retrieving the `meta.json`?